src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -52,6 +53,10 @@ namespace Arcus.Templates.WebApi
 {
     public class Startup
     {
+#if SharedAccessKeyAuth
+        private const string SharedAccessKeyHeaderName = "X-Api-Key";
```

Comment:
This is actually a good reminder as we still have to document this header as a request header for SAS Auth. Can you add this here or a dedicated PR please?

Reply:
Hmm not entirely sure what you mean by that 🤔 .

Reply:
It's not documented in OpenAPI similar to this https://github.com/arcus-azure/arcus.templates/blob/master/src/Arcus.Templates.WebApi/Startup.cs#L166

Reply:
Oh, but isn't this done with the security definition? Like you have to "Authorize" the request in the Swagger UI and there you can fill out the header or query that is needed. No?

Reply:
That's correct but also a header that should be listed as request header/query parameter

Reply:
Anyhow, it's ok

Reply:
But we could still add it?😬

Reply:
Nah it's ok